### ISSUE TITLE
Add `meta` description(s)

### ISSUE THREAD

**Issue by brentryanjohnson** [GSA/smartpay-training#318]
The accessibility and SEO audit revealed that the training site lacks a `meta` description.

Ideally, we would have page-level `meta` descriptions, but this is complicated by the "single-page app" behavior of certain parts of the site. We might need to use something generic for that path, but we should add page-level `meta` descriptions to the lesson pages, and have the layout pick those up in the `meta` tag.

**Comment by JessicaMarine1** [comment]
https://github.com/GSA/smartpay-training/pull/321

**Comment by weiwang-gsa** [comment]
@brentryanjohnson @rebekahperillo is [PR](https://github.com/GSA/smartpay-training/pull/321) is ready for merging into staging, seems it has not been approved yet. 

Also, would you mind letting me now what description text you prefer for following pages:

- Quiz pages, one example should be sufficient for me to figure out how to set up the rest.  e.g. [Travel Training Quiz for Card/Account Holders and Approving Officials](https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/quiz/training_travel/)
- [Full Printable Training](https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/training_travel/printable), can I use "view and print the full Travel Training for Card/Account Holders and Approving Officials" as its description?
- [Training Report](https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/training_reports/)
- [Certificate](https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/certificates/)
- [Admin Page](https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/admin/)

**Comment by brentryanjohnson** [comment]
@rebekahperillo Do you feel comfortable writing these based on your review of the `meta` descriptions I wrote for the lessons? I haven't accessed the full admin and A/OPC flows, so I'm not sure what the best descriptions for those screens would be.

`meta` descriptions are largely for search engine optimization, so frankly they have limited importance for pages behind a "login" (since random searchers won't be able to directly access those pages anyway). But it's good to add them, as they can be used for other purposes down the road if you have them.

**Comment by JessicaMarine1** [comment]
@brentryanjohnson Rebekah is out for the next week. Let's move forward with what we've got and log a new issue for any remaining.

**Comment by brentryanjohnson** [comment]
@JessicaMarine1 @weiwang-gsa Sounds good 👍. 

Wei, I think your drafts 👆 are good for now. If you're blocked on the others until you have some copy for them, I think you can add something generic in the meantime for those screens.

> GSA SmartPay training for agency card and account holders, approving officials, and program coordinators.

**Comment by weiwang-gsa** [comment]
@brentryanjohnson Per our discussion, I merged your PR into staging, and I will add generic meta description for these pages, thank you!

**Comment by weiwang-gsa** [comment]
@brentryanjohnson this is ready for design review: https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/

**Comment by brentryanjohnson** [comment]
@weiwang-gsa It looks like the `meta` description for Card/Account Travel is still missing. The others look good!

![Screenshot of Lesson One Travel Program Overview with Lighthouse scores of 100, 100, 100, and 75, with the alert that document doesn't have a meta description](https://github.com/GSA/smartpay-training/assets/32855580/6f78631e-4f11-434a-894f-a16270f974b6)
![Screenshot of Chrome inspector showing website head tag with empty meta description](https://github.com/GSA/smartpay-training/assets/32855580/d75b42be-1214-401e-bf11-7b1fa4747236)

**Comment by weiwang-gsa** [comment]
@brentryanjohnson thanks for catching it, it was unnecessary collection config file on the travel training block the meta description been generated. Put in PR now to resolve this issue.

**Comment by weiwang-gsa** [comment]
@brentryanjohnson seems the code fix already merge into staging branch, it is ready for your review: https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/

**Comment by JessicaMarine1** [comment]
@A-L33 @rebekahperillo ready for QA

**Comment by A-L33** [comment]
Done!
